include/RAJA/policy/atomic_builtin.hpp

```diff
@@ -242,23 +353,54 @@ RAJA_INLINE char builtin_atomicSub(char* acc, char value)
   return _InterlockedExchangeAdd8(acc, -value);
 }
 
+RAJA_INLINE uint8_t builtin_atomicSub(uint8_t* acc, uint8_t value)
+{
+  uint8_t neg_value = -value;
```

Comment:
Does this work as expected with unsigned integer types?

Reply:
Good catch. I pushed a change.

Reply:
Is there a test where the unsigned value is greater than can be represented in the signed type?